slothy/core/slothy.py

```diff
@@ -472,7 +472,10 @@ def optimize_loop(self, loop_lbl, postamble_label=None, forced_loop_type=None):
 
         early, body, late, _, other_data, loop = \
             self.arch.Loop.extract(self.source, loop_lbl, forced_loop_type=forced_loop_type)
-        loop_cnt = other_data['cnt']
+        try:
```

Comment:
The same `try`-`except` needs to go in `fusion_loop` as in https://github.com/slothy-optimizer/slothy/blob/ce74da9dfc3d3d104665a6849643b1ff60f8cb81/slothy/core/slothy.py#L453. This was removed in fb6fe0db8db40afd0ca2c51c5dc8aedd6bf8ce22.